lib/grinder_sdk.dart

```diff
@@ -544,3 +544,12 @@ class _PubLocalApp extends PubApp {
         script: script, arguments: arguments, runOptions: runOptions);
   }
 }
+
+List<String> _coerceToPathList(filesOrPaths) {
+  if (filesOrPaths is! List) filesOrPaths = [filesOrPaths];
```

Comment:
If `filesOrPaths is File` shouldn't this extract the path e.g. `[filesOrPaths.path]`?